### DISCUSSION TITLE
Is [data-type="equation"] math Font-Size broken?

### DISCUSSION THREAD

**Discussion by DokusLab** [stumpapp/stump#460] [Question]
Is the Font-Size for equations set by stump, @/components/readers/epub/EpubJsReader or somewhere else?

Because it seems to be broken, or am I mistaken?
div[data-type="equation"] math {
    font-size: calc(0.35em + 1vw);
}

Seems to be way off for Desktop, where 1vw would be 113.6px for 1080p width.
![image](https://github.com/user-attachments/assets/cd46690a-09be-4f2f-9a12-b97b29b28e98)

**Comment by aaronleopold** [comment]
> Is the Font-Size for equations set by stump

Stump applies minimal styling overrides [here](https://github.com/stumpapp/stump/blob/main/packages/browser/src/components/readers/epub/themes.ts#L9-L21), I'm sure it's possible that something in there is messing it up. Maybe that `'font-size': 'unset'` bit, if I had to guess.

I don't have any books with equations in them to test this with though. Would you mind [emailing me](mailto:aaronleopold1221@gmail.com) this one? If there are reasons (e.g. copyright) which disallow that, no worries! I can try to find a more open one to debug the issue
